### DISCUSSION TITLE
How to set up SL and TP correctly in Bot for XTB Broker (python)

### DISCUSSION THREAD

**Discussion by Prolobster** [grimme-lab/xtb#1123] [Q&A]
Hello friends,
I've been working on my first trading bot for a few days, it's for the broker xtb and specializes in cfd contracts. I had already gotten the bot to work so far and it had placed orders, but a problem kept cropping up with the calculation of the SL and TP. Unfortunately, in the API documentation and in the xtb playground there is not really anything on the subject of setting and calculating the SL and TP. Does anyone of you have experience with this and could help me so that my SL and TP are calculated correctly and set correctly on the xstation? I would like to thank you in advance for your effort and help

**Comment by awvwgk** [comment]
This seems to be unrelated to this project extended tight binding (xtb) for computational chemistry.
